### DISCUSSION TITLE
Videos, images

### DISCUSSION THREAD

**Discussion by nerkn** [CiviWiki/OpenCiviWiki#1411] [Ideas]
### Idea summary

CiviWiki should be presented

### Further details

I have some ideas and searching. I come this repo, I just want to know what it is. I go to "civi.wiki" and site is only redirection site. also google/youtube shows stupid videos since civi in turkish means nail. I know in python world its hard to keep sites open so I request only video/ or images of sys

**Comment by brylie** [comment]
Thanks, @nerkn! Would you like to help us create an introductory video?

**Comment by Ankitapradhan04** [comment]
Hello,
I'm a beginner in open source and I want to contribute to open-source, so I searched where I could contribute and came across this repo. I've learned Python Programming language so I searched in the python section and came here since no one is assigned. Would you please explain to me how I can contribute and what should I do to solve this issue.

**Comment by letsmakecakes** [comment]
> Hello, I'm a beginner in open source and I want to contribute to open-source, so I searched where I could contribute and came across this repo. I've learned Python Programming language so I searched in the python section and came here since no one is assigned. Would you please explain to me how I can contribute and what should I do to solve this issue.

Maybe this article would help -  https://www.firsttimersonly.com/

**Comment by brylie** [comment]
This issue is about promoting CiviWiki. We would like to make a presentation that explains the CiviWiki concept, project, and how to get involved. So, no programming skills are needed, just design and communication.

**Comment by kevinmenear** [comment]
Hey, this is right up my alley - I can do this and will be glad to make my first open source contribution here. May I begin work on this?

In the meanwhile, I will look around at the CiviWiki resources to get some background information.

**Comment by brylie** [comment]
@kevinmenear, sure thanks! I'll assign you to the issue.

**Comment by kevinmenear** [comment]
Great! I'm on it.

**Comment by blurryface18** [comment]
@kevinmenear is there anything that I can help with , btw I can edit video if you want

**Comment by kevinmenear** [comment]
@blurryface18 are you any good with animation? That'll help. I found this video: https://www.youtube.com/watch?v=-Kd2xeEMb9s. It seems like a good place to start. I'd like to build/run CiviWiki locally to get some visuals. Is this possible? If so - can someone please give me some direction on how to get started? I've crawled through the source code and am at a loss.

**Comment by Emeka-Ugbanu-hub** [comment]
I would love to help with this

**Comment by kevinmenear** [comment]
Hi @Emeka-Ugbanu-hub how would you like to help?

**Comment by Emeka-Ugbanu-hub** [comment]
maybe help with the introductory video @brylie mentioned

**Comment by kevinmenear** [comment]
Ok, great! What skills do you have that you can apply to the video development?

**Comment by Emeka-Ugbanu-hub** [comment]
Javascript

**Comment by kevinmenear** [comment]
Ok - I have had trouble getting the software to run. Maybe you can work on that and get some screen recordings for the video.

**Comment by Emeka-Ugbanu-hub** [comment]
Ok I'll do that, so i have to clone this repo and get it to run

**Comment by Emeka-Ugbanu-hub** [comment]
and what do i screen record

**Comment by kevinmenear** [comment]
Yes - if you can get it to run, screen record anything you can get of using the UI.

**Comment by Emeka-Ugbanu-hub** [comment]
ok i'll do that

**Comment by kevinmenear** [comment]
Great!

**Comment by Emeka-Ugbanu-hub** [comment]
i would have love to help with this but it's a python project

**Comment by kevinmenear** [comment]
Ok, no worries.

**Comment by kevinmenear** [comment]
@brylie can you please give me some help in getting CiviWiki running?

**Comment by brylie** [comment]
@kevinmenear, sure. Start by setting up a CiviWiki development environment as [outlined in the contributing guide](https://github.com/CiviWiki/OpenCiviWiki/blob/develop/CONTRIBUTING.md#development). Let me know if you encounter any specific errors or blockers.

**Comment by Shraddha063** [comment]
Hello, Can I work on this issue to make my first open source contribution??

**Comment by iftekar2** [comment]
Hello!
I have been learning Python and I created some projects. I am interested in contributing to your project. Is there anything that I can contribute to? 

Thank You

**Comment by KevinMenearED** [comment]
Hi iftekar2 - I have been meaning to get to this for ages. Can you try to get CiviWiki running on your local machine and record your screen, showing yourself walking through the tool? This can help with developing media for this project.

**Comment by iftekar2** [comment]
KeninMenearED I would love to help you with that. Can you please tell me how I can run the cod in my local machine? Because this is the first time I am contributing to an open-source project.

**Comment by KevinMenearED** [comment]
For sure - check out brylie's message on this issue from Feb 18. You can find instructions at the link he shared.

**Comment by ritsvishalcs** [comment]
I'm new to open source and I want to contribute can anyone help me please

**Comment by brylie** [comment]
What are your skills and interests that bring you to open source?
